Yatabar.lua

```diff
@@ -1128,9 +1129,11 @@ end
 
 function Yatabar:HidePopups()
 	for element, idx in pairs(Yatabar.orderElements) do
-		Yatabar["TotemHeader"..element]:Execute([[
+		if Yatabar["TotemHeader"..element] ~= nil then
```

Comment:
I would do the check for nil outside of the execute, so only on Execute is necessary